### DISCUSSION TITLE
Еrror with termination when generating on presets other than the standard one

### DISCUSSION THREAD

**Discussion by vaulthunt3r** [lllyasviel/Fooocus#3181] [Q&A] [answered]
![cmd_7vkUf02t8q](https://github.com/lllyasviel/Fooocus/assets/160418681/17ee898d-fae8-45b6-a68a-45f5caa18926)
When I change the preset from “initial” to others (I often used anime), when I click the “Generate” button, an error occurs, without any instructions in the console. It’s the same on other forks, the problem is suspected to be outside the fooocus folder. Python drivers have not been updated. Where to look, what to reinstall?

**Comment by mashb1t** [comment, marked as answer]
@vaulthunt3r this is most likely an issue with your system swap. Please find instructions how to solve this in the https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md.

**Reply by vaulthunt3r** [reply]
The nuance is that everything is as it was originally, nothing has changed, the drivers have not been updated, i will have to reinstall the OS

**Reply by mashb1t** [reply]
Please double-check swap before reinstalling the OS as this might not be necessary.

**Reply by vaulthunt3r** [reply]
![SystemPropertiesAdvanced_qzQ03ZYtjr](https://github.com/lllyasviel/Fooocus/assets/160418681/4696e110-de43-453a-9f28-e693a342bf72), Same thing on the new system.

**Reply by mashb1t** [reply]
@vaulthunt3r try deactivating automatic allocatoon and put the swap file on the same drive as Fooocus

**Reply by vaulthunt3r** [reply]
Hurray, this solved the problem, thanks my friend :), made me a little happy.

**Reply by vaulthunt3r** [reply]
what could this mean? I don't quite understand what the problem is. Before this it was on automatic and there were no problems. :)

**Reply by mashb1t** [reply]
i can only assume that Windows relocated the swap file due to high disk usage or because of potential performance improvements when using another drive.
